doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+=======================================
+Harmonic analysis via the krylov method
+=======================================
+
+Introduction
+============
+You can use the frequency-sweep Krylov method for a high performance solution of forced-frequency simulations 
+in acoustic or single-field structural analyses.
+
+Similar to the full harmonic analysis, the frequency sweep Krylov method also uses the full system matrices 
+to compute the harmonic response. While the full method solves the full system matrices at every frequency point 
+in the frequency range, the frequency-sweep Krylov method performs the following steps to approximate the response 
+across the frequency range :
+
+* Builds a Krylov subspace set of vectors at the frequency value in the middle of the requested frequency range,
+* Reduces the system matrices and loading on the entire frequency range,
+* Solves the reduced system, and
+* Expands the results back to compute the harmonic response. 
+
+MAPDL provides the following methods to implement a harmonic analysis using the Krylov Method:
+
+#. Via Mechanical APDL commands.
+#. Via macros, a feature described in *Frequency-Sweep Harmonic Analysis via the Krylov Method* in the *Structural Analysis Guide*.
+#. PyMAPDL implementation of the macros, a feature described here.
+
+Assumptions
+-----------
+The following assumptions are made in the Krylov solution obtained using this method :
```

Comment:
```suggestion
The following assumptions are made in the Krylov solution obtained using this method:
```

Reply:
The sentence is line 28 is not clear to me. Does "using this method" refer to the Kylov method? Can the sentence be rewritten to "The following assumptions are made when using the Kylov method to obtain the solution:"